src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -61,6 +61,7 @@
 
     public enum Feature {
         SWITCH_PATTERN_MATCHING,
+        DECONSTRUCTION_PATTERNS,
```

Comment:
The spec and JEP talks about record patterns - I think we should follow the correct name here

Reply:
yup, I agree